scripts/mbedtls_dev/crypto_knowledge.py

```diff
@@ -190,7 +190,7 @@ def is_valid_for_signature(self, usage: str) -> bool:
            signitute type.
 
         """
-        # This is just temporaly solution for the implicit usage flags.
+        # This is just temporary solution for the implicit usage flags.
```

Comment:
can we fix the grammar at the same time?
```suggestion
        # This is just a temporary solution for the implicit usage flags.
```